app/src/main/java/com/example/playlistmaker3/TrackView.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.playlistmaker3
+
+import android.content.Context
+import android.util.TypedValue
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.engine.DiskCacheStrategy
+import com.bumptech.glide.load.resource.bitmap.CenterCrop
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners
+import com.bumptech.glide.request.RequestOptions
+
+
+ class TrackView(track: View,private val context: Context) : RecyclerView.ViewHolder(track) {
```

Comment:
```suggestion
 class TrackViewHolder(track: View,private val context: Context) : RecyclerView.ViewHolder(track) {
```
Чтобы не путать с обычной вью